dl/src/main/scala/com/intel/analytics/bigdl/models/rnn/Train.scala

```diff
@@ -85,17 +85,18 @@ object Train {
       }
 
       Engine.init(1, param.coreNumber, false)
+      val criterion = new CrossEntropyCriterion[Float](squeezeFlag = true)
```

Comment:
Pass this criterion to Loss validation method. Inside the validation method, it will clone it when the apply is called.